### PR TITLE
Update OC Compatiblity

### PR DESCRIPTION
**Pull request by mlch911**

### CODE REVIEW

File: Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift [hunk 1]
```diff
@@ -28,11 +28,17 @@ public final class CompatibleAnimation: NSObject {
...
+    super.init()
+    self.animation = animation
+  }
+
   // MARK: Internal
 
-  var animation: LottieAnimation? {
-    LottieAnimation.named(name, bundle: bundle, subdirectory: subdirectory)
-  }
+  private(set) lazy var animation = LottieAnimation.named(name, bundle: bundle, subdirectory: subdirectory)
```
Comment (calda) [inline Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift:41]: Instead, what if we just make this:

```suggestion
  var animation: LottieAnimation
```

and update the existing `init` to just:

```swift
  public init(
    name: String,
    subdirectory: String? = nil,
    bundle: Bundle = Bundle.main)
  {
    animation = LottieAnimation.named(name, bundle: bundle, subdirectory: subdirectory)
    super.init()
  }
```

Comment (mlch911) [reply Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift:41]: No problem

**Comment by calda** [comment]
Looks like the build is failing -- also make sure to run `bundle exec rake format:swift` to satisfy the linter. (Lottie follows Airbnb's [Swift Style Guide](https://github.com/airbnb/swift)).

File: Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift [hunk 2]
```diff
@@ -46,6 +47,12 @@ public final class CompatibleAnimation: NSObject {
   private let bundle: Bundle
 }
 
+extension LottieAnimation {
+  public func ocCompatible() -> CompatibleAnimation {
```
Comment (calda) [inline Sources/Public/iOS/Compatibility/CompatibleAnimationView.swift:51]: Is this necessary? I'd prefer to keep the change focused on the new `init`

**Comment by calda** [comment]
Still seeing build failures

**Comment by calda** [comment]
Since the build is failing I'm going to go ahead and close this PR for now. If you'd like, feel free to re-open the PR after fixing the build errors!

**Pull request closed without merging**